wagtailio/core/blocks.py

```diff
@@ -566,4 +574,4 @@ class ContentStoryBlock(blocks.StreamBlock):
     comparison_table = ComparisonTableBlock()
 
     class Meta:
-        template = "patterns/components/streamfields/content_story_block.html"
+        template = "patterns/components/content_story_block.html"
```

Comment:
Suggestion: Could we move the streamfield block into streamfields? I appreciate that this is possibly not the most semantic place but I think the grouping is more logical.

Reply:
Great! Done in d79fba7